tsl/src/debug.c

```diff
@@ -62,25 +60,16 @@ static const char *reloptkind_name[] = {
 	[RELOPT_BASEREL] = "BASEREL",
 	[RELOPT_JOINREL] = "JOINREL",
 	[RELOPT_OTHER_MEMBER_REL] = "OTHER_MEMBER_REL",
-#if PG11_GE
 	[RELOPT_OTHER_JOINREL] = "OTHER_JOINREL",
-#endif
 	[RELOPT_UPPER_REL] = "UPPER_REL",
-#if PG11_GE
 	[RELOPT_OTHER_UPPER_REL] = "OTHER_UPPER_REL",
-#endif
 	[RELOPT_DEADREL] = "DEADREL",
 };
 
 static const char *upperrel_stage_name[] = {
-	[UPPERREL_SETOP] = "SETOP",
-#if PG11_GE
-	[UPPERREL_PARTIAL_GROUP_AGG] = "PARTIAL_GROUP_AGG",
-#endif
-	[UPPERREL_GROUP_AGG] = "GROUP_AGG",
-	[UPPERREL_WINDOW] = "WINDOW",
-	[UPPERREL_DISTINCT] = "DISTINCT",
-	[UPPERREL_ORDERED] = "ORDERED",
+	[UPPERREL_SETOP] = "SETOP",			[UPPERREL_PARTIAL_GROUP_AGG] = "PARTIAL_GROUP_AGG",
```

Comment:
Formatting here looks odd, is this after `make format`?

Reply:
make format messes this one up for some reason and doesnt respect the formatting